support-frontend/assets/helpers/abTests/abtest.js

```diff
@@ -201,6 +202,14 @@ function getParticipations(
       return;
     }
 
+    if (test.urlParticipationOnly) {
```

Comment:
I feel like "only" somehow makes it ambiguous.
Can we just say `participationMethod` and it's either `url` or `standard` (or something) then it's clear it's either/or